test/functional/app/api/test_credit_transfer_api.py

```diff
@@ -36,7 +36,13 @@ def test_create_credit_transfer(test_client, authed_sempo_admin_user, create_tra
     if recipient_transfer_accounts_ids:
         recipient_transfer_accounts_ids = [recipient_transfer_accounts_ids]
 
+    # Hack to prevent previous tests from causing future tests to fail
+    # TODO: Change design of entire testing process to enable quick setup of blockchain etc, but not have tests so path dependent
```

Comment:
Transfers from previous tests are persisted because the db isn't torn down, which results in this crap